### DISCUSSION TITLE
Tracking Japanese Localization Progress

### DISCUSSION THREAD

**Discussion by naonishijima** [cncf/glossary#2111] [General]
Discussion to manage and track Japanese localization progress.

## Information: Japanese localization
- https://github.com/cncf/glossary/issues/1422
  - We welcome contributors.
  - Currently, there are 3 Japanese contents approvers.
- We use a development branch (https://github.com/cncf/glossary/tree/dev-ja).
  - Contributors need to use dev-ja as the base branch, and the target branch of a PR should be dev-ja.
- For more details and any information, please ask on the CNCF slack channel: [#glossary-localization-japanese](https://app.slack.com/client/T08PSQ7BQ/C057F81GFUG).
- We track Japanese Localization Progress in [Projects]( https://github.com/cncf/glossary/projects/13)

## Contribution procedure

1. Check for unassigned words in [Japanese Localization Progress]( https://github.com/cncf/glossary/projects/13).
    - Assign yourself to a word that no one else has assigned.
    - Please take charge of one PR at a time. 
    - If not a word in Projects, create an issue and assign it yourself.
    - Associate the opened Issue with Japanese Localization Progress at Projects.
2. Update branches as the latest
3. Copy content/en/(your assigned term).md to content/ja/(your assigned term).md
4. Localize your assigned term
5. Localize PR only requests their language branch(into cncf:dev-ja from your_repo:dev-ja)
    - Contributors need to use dev-ja as the base branch, and the target branch of a PR should be dev-ja.
    - Please use concise and informative English PR title. (ex: [ja] Localize cloud computing).
    - Add lang/ja labels
6. Assign Japanese approvers @kaitoii11 @inductor @naonishijima as reviewers. 
7. Japanese approvers will review the PR.
8. Update the PR according to reviewers' comments (or discuss before update).

## Localization approvers/reviewers
Any questions or help, just talk with us (here or by slack):
@inductor 
@kaitoii11 
@naonishijima 

## Taks list
We need at least 10 terms in Japanese before go to live (v1), below are some suggested terms to work (but not limited):

| # | File to localize | Assignee | Ref(#PR) | 
| - | - | - | - |
| conf | content/ja/_TEMPLATE.md | @naonishijima | #2088 |
| conf | content/ja/_index.md | @naonishijima | #2088 |
| conf | contribute/_index.md | @naonishijima | #2088 |
| conf | style-guide/_index.md | @naonishijima | #2088 |
| conf | i18n/ja.toml | @naonishijima | #2088 |
| 1 | content/ja/cloud-computing.md | @naonishijima | #2088 |

**Comment by Okabe-Junya** [comment]
We (almost) completed ja l10n, so let me close discussion. if you need some help, please ask in [#glossary-localization-japanese](https://cloud-native.slack.com/archives/C057F81GFUG)
